### PR TITLE
feat: we can use kubectl logs and docker logs review apisix container's logs.

### PR DESCRIPTION
**Pull request by sober-wang**
we can use `kubectl logs` and `docker logs` review apisix container's logs

```shell
kubectl -n <YOUR_NAMEPSACES> logs <APISIX_POD>
docker logs <APISIX_CONTAINERS>
```

### CODE REVIEW

File: alpine/Dockerfile [hunk 1]
```diff
@@ -40,6 +40,10 @@ COPY --from=production-stage /usr/bin/apisix /usr/bin/apisix
 
 ENV PATH=$PATH:/usr/local/openresty/luajit/bin:/usr/local/openresty/nginx/sbin:/usr/local/openresty/bin
 
+# apisix's logs will output the stdout and stderr. we can use commend `kubectl logs` and `docker logs` review apisix's logs
```
Comment (imjoey) [inline alpine/Dockerfile:43]:
```suggestion
# APISIX's logs will output the stdout and stderr. We can use command `kubectl logs` and `docker logs` to review APISIX's logs
```

**Review by imjoey — CHANGES_REQUESTED** [review]

**Comment by imjoey** [comment]
Hi @sober-wang , thank you. Could you please add relevant test cases in CI to check the logs output? That would make the PR much reliable. Thanks.

**Comment by sober-wang** [comment]
I know your mean's. But can you help me ,how to create a test cases in CI.

**Comment by imjoey** [comment]
> I know your mean's. But can you help me ,how to create a test cases in CI.

Hi @sober-wang , you could try login via curl and then see if `docker logs` contains relevant logs,  please refer to https://github.com/apache/apisix-dashboard/blob/8e1854704d5545f7574fb69fe2710f19871279fd/.github/workflows/backend-unit-test.yml#L55-L61 for example.

BTW: I checked out your code in my local MacOS, but unfortunately I was not able to see continuous logs output via `docker logs`. Of course, perhaps there's something wrong with my environment. 😄 So additional checks in CI would be better.

**Comment by sober-wang** [comment]
> > I know your mean's. But can you help me ,how to create a test cases in CI.
> 
> Hi @sober-wang , you could try login via curl and then see if `docker logs` contains relevant logs, please refer to https://github.com/apache/apisix-dashboard/blob/8e1854704d5545f7574fb69fe2710f19871279fd/.github/workflows/backend-unit-test.yml#L55-L61 for example.
> 
> BTW: I checked out your code in my local MacOS, but unfortunately I was not able to see continuous logs output via `docker logs`. Of course, perhaps there's something wrong with my environment. 😄 So additional checks in CI would be better.

maybe , you should do something look like this
```shell
cd  apisix-docker/centos
docker build -t newapisix-image:20210218 
docker run -d -p 9080:9080 --name=newapisix newapisix-image:20210218
# next used 
docker logs newapisix 
# you'll see the apisix logs in stdout
```

**Comment by gxthrj** [comment]
I added a CI in #133 , we can merge this PR after #133 merged.

**Comment by gxthrj** [comment]
We can continue to review this pr, @sober-wang Need to modify the document format as @imjoey  suggest.
As #133  has been merged, we need to rebase master.

**Comment by spacewander** [comment]
Solved.

**Pull request closed without merging**